File: ios/Sources/NapSspIOSSample/AdEventLogger.swift
import Foundation

enum AdEventLogger {
    static func request(format: String, id: String) { print("NapSsp request format=\(format) id=\(id)") }
    static func loaded(format: String, id: String) { print("NapSsp loaded format=\(format) id=\(id)") }
    static func displayed(format: String, id: String) { print("NapSsp displayed format=\(format) id=\(id)") }
    static func clicked(format: String, id: String) { print("NapSsp clicked format=\(format) id=\(id)") }
    static func failed(format: String, id: String, reason: String) { print("NapSsp failed format=\(format) id=\(id) reason=\(reason)") }
}

File: android/app/src/main/java/com/gwangy/nassspandroidsample/AdEventLogger.kt
package com.gwangy.nassspandroidsample

object AdEventLogger {
    fun request(format: String, id: String) = println("NapSsp request format=$format id=$id")
    fun loaded(format: String, id: String) = println("NapSsp loaded format=$format id=$id")
    fun displayed(format: String, id: String) = println("NapSsp displayed format=$format id=$id")
    fun clicked(format: String, id: String) = println("NapSsp clicked format=$format id=$id")
    fun failed(format: String, id: String, reason: String) = println("NapSsp failed format=$format id=$id reason=$reason")
}
